googleMap.js

```diff
@@ -0,0 +1,226 @@
+/*
+	This object handles the map coming from the Google Maps API
+
+*/
+var GoogleMap = function() {
+	var midLat = (APP.city.lat.min + APP.city.lat.max) / 2;
+	var midLong = (APP.city.long.min + APP.city.long.max) / 2;
+
+	// var nycLatLong = new google.maps.LatLng(40.7435776, -73.98629);
+	this.mapOptions = {
+		zoom: 13,
+		center: new google.maps.LatLng(midLat, midLong)
+	};
+
+	this.map = null;
+	this.markers = [];
+};
+
+GoogleMap.prototype.initialize = function() {
+	this.map = new google.maps.Map(document.getElementById('map-canvas'), this.mapOptions);
+	this.setStyle();
+};
+
+GoogleMap.prototype.plotLocations = function(imageObjects) {
+	$.each(imageObjects, function(i, imageObject) {
+		var lat = imageObject.location.latitude;
+		var long = imageObject.location.longitude;
+		var latLong = new google.maps.LatLng(lat, long);
+
+		var existingMarker = this.checkForExistingMarker(latLong);
+		if (!existingMarker) {
```

Comment:
If you're using an `if...else`, it's good style to use the form

```javascript
if (condition) {
  // truthy stuff
} else {
  // falsy stuff
}
```

Otherwise, the reader needs to think about the double negative, which is confusing.